src/reducers/summaryReducer.js

```diff
@@ -17,7 +18,7 @@ export default (state = initialState, action) => {
     case SUMMARY_RECEIVED:
       return {
         ...state,
-        ...action.summaries,
+        data: { ...action.summaries },
```

Comment:
Should this be `data: { ...state.summaries, ...action.summaries }` so that we don't lose data that is already in the reducer? Or does it not really matter since most of the data will be held the localStorage cache?